migrations/1_create_asset_allocations_up.sql

```diff
@@ -0,0 +1,8 @@
+    CREATE TABLE asset_allocations(
+        id SERIAL PRIMARY KEY,
+        allocated_by varchar(100) NOT NULL,
+        isactive boolean NOT NULL,
+        FOREIGN KEY(user_id),
```

Comment:
is this the way to define a foreign key in postgres?

Reply:
FOREIGN KEY(fk_columns) yes this is another way to define foreign key in postgre

Reply:
this is throwing an error while creating table

Reply:
here, you should also define the reference column name of parent table